legal-api/src/legal_api/services/filings/validations/alteration.py

```diff
@@ -97,3 +98,15 @@ def company_name_validation(filing):
                         'path': legal_name_path})
 
     return msg
+
+
+def type_change_validation(filing):
+    """Validate type change."""
+    msg = []
+    legal_type_path: Final = '/filing/alteration/business/legalType'
+    # you must alter to a bc benefit company
+    if get_str(filing, legal_type_path) != 'BEN':
+        msg.append({'error': babel('Your business type has not been updated to a BC Benefit Company.'),
```

Comment:
Usually it's good to say why the error occurred (esp as we display the error text in the UI). Is this message approved by PO/UX?

Reply:
It's from the ticket: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/7100 :)